include/openssl/err.h

```diff
@@ -339,7 +339,17 @@ enum {
   ERR_LIB_HKDF,
   ERR_LIB_TRUST_TOKEN,
   ERR_LIB_USER,
-  ERR_NUM_LIBS
+  ERR_NUM_LIBS,
+  ERR_LIB_PKCS12,
+  ERR_LIB_DSO=37,
+  ERR_LIB_OSSL_STORE=44,
+  ERR_LIB_FIPS,
+  ERR_LIB_CMS,
+  ERR_LIB_TS,
+  ERR_LIB_CT=50,
+  ERR_LIB_ASYNC,
+  ERR_LIB_KDF,
+  ERR_LIB_SM2
```

Comment:
in light of [this commit](https://github.com/aws/aws-lc/commit/6bdd10042ea5e9015bb1ecf740b76a29ea7c0302), you'll need to convert these into macros. do we know why there are explicit values assigned to some of these enum entries?

Reply:
Ah, I'll make that change. 

There were many other error codes in OpenSSL that I have not added here, only those required by NodeJS. By providing explicit values I've tried to maintain that same values for those error codes across libraries. However, the values shouldn't matter so I can also make them numerically ascending if we prefer that

Reply:
https://github.com/openssl/openssl/blob/5677992679b38950c6a0c3775fd57378e1879ba5/include/openssl/err.h.in#L113